### PR TITLE
Bound probe to any interface

### LINKED ISSUE #57: Allow probe port to be bound to all interfaces (and not just localhost) during development
The current implementation is perfectly working in production scenarios.
However, during development (especially in container-based development) it would be helpful make it possible to bind the probe application to all interfaces, too. That would make it more convenient to develop probe-based check code.

That bothered me when using Hurricane with Gefyra while forwarding the probe ports. They are not answered to Kubernetes since the probe request does not originate from localhost (as it would be running in the cluster) but is proxied.

### PR DESCRIPTION
**Pull request by vvvityaaa**
Fixes #57

### CODE REVIEW

File: hurricane/management/commands/consume.py [hunk 1]
```diff
@@ -35,13 +35,15 @@ class Command(BaseCommand):
         - ``--startup-probe`` - the exposed path (default is /startup) for probes to check startup
         - ``--readiness-probe`` - the exposed path (default is /ready) for probes to check readiness
         - ``--liveness-probe`` - the exposed path (default is /alive) for probes to check liveness
+        - ``--interface`` - set a host name for probe server
```
Comment (Schille) [inline hurricane/management/commands/consume.py:38]: I'd not make this an option. Bind the probe port always to all interfaces.

**Comment by Schille** [comment]
I'd say don't have another option for that. It's getting quite complicated. Please just always bind the probe port to all available interfaces and this should be good for the moment.

**Comment by vvvityaaa** [comment]
@Schille without any additional settings, tornado binds to all interfaces, it is mentioned in this [official tornado issue](https://github.com/tornadoweb/tornado/issues/2718#issuecomment-517488766).

Here is also output from the hurricane server, that shows, that it can bind to the local ip
```
(venv) viktor@t490s:~/PycharmProjects/hurricane$ python manage.py serve
System check identified some issues:

WARNINGS:
testapp.Choice: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
testapp.Question: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
        HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
2022-04-22 09:25:03,035 INFO     hurricane.server.general Tornado-powered Django web server
2022-04-22 09:25:03,035 INFO     hurricane.server.general Starting probe application running on port 8001 with route liveness-probe: /alive, readiness-probe: /ready, startup-probe: /startup
2022-04-22 09:25:03,037 INFO     hurricane.server.general Starting HTTP Server on port 8000
2022-04-22 09:25:03,037 INFO     hurricane.server.general Startup time is 0.0024394989013671875 seconds
2022-04-22 09:29:33,554 INFO     hurricane.server.access 200 GET / (192.168.222.21) 4.11ms
2022-04-22 09:29:33,756 ERROR    hurricane.server.access 500 GET /favicon.ico (192.168.222.21) 6.62ms
2022-04-22 09:30:09,760 INFO     hurricane.server.access 200 GET / (127.0.0.1) 3.61ms
```

**Comment by tschale** [comment]
@vvvityaaa @Schille Is this ready to be deployed?

**Comment by vvvityaaa** [comment]
@tschale no, my last comment was only the info, that this functionality is inbuilt in the tornado. Basically, this was just for the info, the PR should be closed without merging.

**Pull request closed without merging**